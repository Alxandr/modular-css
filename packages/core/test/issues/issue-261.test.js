"use strict";

const { find, cwd } = require("test-utils/fixtures.js");

const Processor = require("../../processor.js");

describe("/issues", () => {
    describe("/261", () => {
        it("should allow colons in rules that also use :external()", async () => {
            var processor = new Processor({ cwd });
            
            await processor.file(find("261/2.css"));

            const result = await processor.output();
            
            expect(result.css).toMatchSnapshot();
        });
    });
});
