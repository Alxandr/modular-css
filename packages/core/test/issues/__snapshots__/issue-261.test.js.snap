// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/issues /261 should allow colons in rules that also use :external() 1`] = `
"/* 261/1.css */
.mcaa907a70_a {
    color: red;
}
/* 261/2.css */
.mc1efe3491_b:hover .mcaa907a70_a {
    color: blue;
}
"
`;
